rmgpy/pdep/sls.py

```diff
@@ -101,3 +101,25 @@ def solve_me_fcns(f,jac,M,p0,t):
     sol = de.solve(prob,solver=de.CVODE_BDF(),abstol=1e-16,reltol=1e-6)
     return sol
 
+def states_to_configurations(network,indices,state,exclude_association=False):
```

Comment:
Could you add blanks after commas

Reply:
added

Reply:
Could you squash the commit where you add the blanks? Thanks

Reply:
This shouldn't exist in the first place anymore